cmd/env/setcustom.go

```diff
@@ -53,7 +53,7 @@ var SetCustCmd = &cobra.Command{
 func init() {
 	SetCustCmd.Flags().StringVarP(&memberName, "name", "n",
 		"", "Member Name, example Service Account Name")
-	SetCustCmd.Flags().StringVarP(&role, "role", "r",
+	SetCustCmd.Flags().StringVarP(&role, "role", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `role` is breaking change right?